webpack.config.js

```diff
@@ -40,7 +40,8 @@ const config = {
     },
     resolve: {
         alias: {
-            root: path.resolve('./src')
+            Root: path.resolve('./src'),
+            Helpers: path.resolve('./src/helpers')
```

Comment:
I noticed you don't actually use this alias.

Reply:
It was removed earlier by mistake. I added it again.